addons/portal/wizard/portal_wizard.py

```diff
@@ -40,23 +40,37 @@ def _default_user_ids(self):
                 # make sure that each contact appears at most once in the list
                 if contact.id not in contact_ids:
                     contact_ids.add(contact.id)
-                    in_portal = False
-                    if contact.user_ids:
-                        in_portal = self.env.ref('base.group_portal') in contact.user_ids[0].groups_id
                     user_changes.append((0, 0, {
                         'partner_id': contact.id,
                         'email': contact.email,
-                        'in_portal': in_portal,
```

Comment:
While we are at it we could clean a bit this default-get-computation, and have
  * a partner_ids field (default coming from active_ids if not given through default_partner_ids)
  * then user_ids being a computed field based on partner_ids ?

Could be a bit more "modern", what do you think ?

Reply:
@tde-banana-odoo But we will still have a method `default_get_partner_ids` and a new compute method? (so I don't think it will make the code cleaner)

Reply:
Yes, but default get would be simple: if not other values, take active_ids . And logic would be moved onto a real compute field. Default should not be the place where we compute things. Globally it would lead to same code, just split into a default (support active_ids) and a compute (implement logic from partners to portal users).